src/Files.App/UserControls/TabBar/TabBar.xaml.cs

```diff
@@ -139,12 +142,24 @@ private void TabHoverSelected(object sender, object e)
 
 		private void TabView_TabDragStarting(TabView sender, TabViewTabDragStartingEventArgs args)
 		{
+			cancelTagDrag = false;
 			var tabViewItemArgs = (args.Item as TabBarItem).NavigationParameter;
 			args.Data.Properties.Add(TabPathIdentifier, tabViewItemArgs.Serialize());
 			args.Data.RequestedOperation = DataPackageOperation.Move;
 
 			InteropHelpers.GetCursorPos(out dragStartPoint);
 			dragStartTime = DateTimeOffset.UtcNow;
+
+			// Force focus in order that the keydown event works
+			Focus(FocusState.Programmatic);
+			PreviewKeyDown += (s, e) =>
```

Comment:
Is there a way to unsubscribe?

Reply:
Alternatively we can create a separate method, but we'd need a way to access `args.Cancel`.